src/tests/bancroft.rs

```diff
@@ -26,9 +28,14 @@ fn test() {
         IonoComponents::Unknown,
         TropoComponents::Unknown,
     );
-    let st =
-        OrbitalState::from_position((24170352.34904016, -16029029.85873581, -5905924.153143198));
-    cd0.set_state(st);
+    let orbit = Orbit::from_position(
+        24170352.34904016 / 1.0E3,
+        -16029029.85873581 / 1.0E3,
+        -5905924.153143198 / 1.0E3,
+        Epoch::from_str("2020-06-20T00:00:00 GPST").unwrap(),
+        EARTH_J2000, //wrong ! wgs84
```

Comment:
Consider using the [ITRF93](https://docs.rs/anise/latest/anise/constants/frames/constant.EARTH_ITRF93.html) frame or IAU_EARTH_FRAME here. In one application that I have, we use the ITRF93 frame and it matches with some internal verification.

Reply:
The two key structures to my apps, are `Solver` within this very crate, and `QcContext` in the rinex libs.
Both of them now have an Almanac and a Frame defintion. The frame needs to be ECEF in these kind of applications.
I set it to ITRF93 because you said this before in previous conversations.

Is it safe to use ITRF93 anytime I need an ECEF ?
What is the difference between IAU and all the other ECEF (ITRF93, EARTH_J2000..) ?

Reply:
thank you very much for taking the time to explain once again.
It is natural to require remote access for ITRF models, but then I don't understand why the current form seems to work.

1. ANISE should throw an error instead, when I try to obtain the ITRF93 model while the Almanac was built with `until_2035`. Are we sure what's happening inside your core is what you intend ?
2. I use EARTH_J2000 lazily, because I have seen you use it in several places. For example when obtaining unit vectors (r_hat) between Earth and Sun, we used EARTH_J2000 and SUN_J2000. The bottom line I guess, is I have no idea if some operations may require inertial frames while others may require fixed frames.

In my mind, at least that's what I foresaw, I'd like to propose an interface for either "autonomous" deployment (no remote access) limited to basic ECEF models, and "cloud" deployment, where the best models can be used

Reply:
ANISE will throw an error if you try to perform any rotation or translation from the ITRF93 frame to any other frame and the ITRF93 frame is not loaded. Let me know if that isn't the case, because I would be very surprised (and it would be a bug).

In astrodynamics, we tend to use J2000 or the International Celestial Reference Frame (ICRF) which are both inertial frame, which is why it's used everywhere in the documentation of the ANISE or Nyx. You want to use inertial frames when you're concerned with the position of your vehicle with respect to the stars or planets. When you're concerned with the latitude or longitude, you need to express the state of the spacecraft with respect to the celestial object for which you're computing that latitude and longitude. That's because the latitude of your vehicle depends on the exact orientation of the Earth (for example) at a given time. In other words, whenever you need information about a spacecraft that is relative to the orientation of a celestial object with respect to the stars, you need to use the body fixed frame.

Reply:
I will have to save this answer somewhere 😄 

Ok so because I'm not 100% finished here and the rotations are not applied yet (-> small 10m errors), I have not run into the problem (in other words, all the calculations I do are feasible). 
I am about to implement the rotations, I will just leave it as is to see if we indeed fall into the coordinates system error.

I will try to "rework" my coordinates system definition at the same time, but I'm not sure how to do that yet